clients/admin-ui/src/features/config-wizard/ConfigWizardWalkthrough.tsx

```diff
@@ -90,47 +66,6 @@ const ConfigWizardWalkthrough = () => {
               <ScanResults />
             </Box>
           ) : null}
-          {/* These steps should only apply if you're creating systems manually */}
```

Comment:
Big changes to what was once the "config wizard" here: in the past, the steps were

1. organization form
2. add systems
3. authenticate scanner
4. scan results
5. manually create

These steps were stored in redux, but they have been making less and less sense as we change up the add system flow (i.e. why would manually create come after scanning results?). this removes step 5 in favor of routing to a new page meant for manually creating systems `/add-systems/new` so we can remove all of step 5